pom.xml

```diff
@@ -150,129 +152,179 @@ under the License.
                 </dependency>
 	</dependencies>
 
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.codehaus.mojo</groupId>
-				<artifactId>exec-maven-plugin</artifactId>
-				<executions>
-					<execution>
-						<id>run-benchmarks</id>
-						<goals>
-							<goal>exec</goal>
-						</goals>
-					</execution>
-				</executions>
-				<configuration>
-					<classpathScope>test</classpathScope>
-					<executable>java</executable>
-					<arguments>
-						<argument>-classpath</argument>
-						<classpath/>
-						<argument>org.openjdk.jmh.Main</argument>
-						<argument>-e</argument>
-						<argument>org.apache.flink.benchmark.full.*,org.apache.flink.state.benchmark.*</argument>
-						<argument>-rf</argument>
-						<argument>csv</argument>
-						<argument>.*</argument>
-					</arguments>
-				</configuration>
-			</plugin>
+	<profiles>
+		<profile>
+			<id>travis</id>
+			<activation>
+				<activeByDefault>false</activeByDefault>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>exec-maven-plugin</artifactId>
+						<version>${maven.exec.version}</version>
+						<executions>
+							<execution>
+								<id>test-benchmarks</id>
+								<phase>test</phase>
+								<goals>
+									<goal>exec</goal>
+								</goals>
+							</execution>
+						</executions>
+						<configuration>
+							<skip>${skipTests}</skip>
+							<classpathScope>test</classpathScope>
+							<executable>java</executable>
+							<arguments>
+								<argument>-classpath</argument>
+								<classpath/>
+								<argument>org.openjdk.jmh.Main</argument>
+								<!--shouldFailOnError-->
+								<argument>-foe</argument>
+								<argument>true</argument>
+								<!--speed up tests-->
+								<argument>-f</argument>
+								<argument>1</argument>
+								<argument>-i</argument>
+								<argument>1</argument>
+								<argument>-wi</argument>
+								<argument>0</argument>
+								<argument>-rf</argument>
+								<argument>csv</argument>
+								<argument>.*</argument>
+							</arguments>
+						</configuration>
+					</plugin>
+
+					<plugin>
```

Comment:
Could we deduplicate/share `maven-compiler-plugin` and `avro-maven-plugin` across both profiles?